### PR TITLE
Add tab-completer for /spawnmob command

### PR DESCRIPTION
**Pull request by montlikadani**

### CODE REVIEW

**Comment by montlikadani** [comment]
~~https://travis-ci.com/EssentialsX/Essentials/builds/118431722#L1916
wut?~~

**Review by Omio — COMMENTED** [review]
Beyond a mildly minimal memory impact, is this really going to change a bunch, bar for compile time?

**Comment by montlikadani** [comment]
> Beyond a mildly minimal memory impact, is this really going to change a bunch, bar for compile time?

Because of many tests, there is so much to commit because Travis didn't accept it.

**Review by Ichbinjoe — DISMISSED** [review]

File: Essentials/src/com/earth2me/essentials/commands/Commandspawnmob.java [hunk 1]
```diff
@@ -59,4 +60,13 @@ public void run(final Server server, final CommandSource sender, final String co
         final User target = getPlayer(ess.getServer(), args, 2, true, false);
         SpawnMob.spawnmob(ess, server, sender, target, mobParts, mobData, mobCount);
     }
+
+    @Override
+    protected List<String> getTabCompleteOptions(Server server, User user, String commandLabel, String[] args) {
+        if (args.length == 1) {
+            return Lists.newArrayList(SpawnMob.mobParts(args[0]));
+        } else {
+            return java.util.Collections.emptyList();
```
Comment (pop4959) [inline Essentials/src/com/earth2me/essentials/commands/Commandspawnmob.java]: To be consistent with the rest of the Essentials codebase, I suggest reverting this back to not using a fully qualified package name.

**Review by pop4959 — CHANGES_REQUESTED** [review]

**Review by pop4959 — APPROVED** [review]

**Review by Ichbinjoe — APPROVED** [review]

**Pull request merged**